firmware/robot2015/hw-test/CMakeLists.txt

```diff
@@ -36,15 +36,15 @@ if(HW_TEST_UNIT AND NOT(HW_TEST_UNIT MATCHES "default"))
 
     # custom target for creating a .bin file from an elf binary
     add_custom_target(robot2015-test
-        arm-none-eabi-objcopy -O binary robot2015_test_ELF ${TEST_UNIT_BIN_FILE}
-        COMMAND cp ${CMAKE_CURRENT_BINARY_DIR}/${TEST_UNIT_BIN_FILE} ${PROJECT_SOURCE_DIR}/run/
+        ${ARM_OBJCOPY} -O binary robot2015_test_ELF ${TEST_UNIT_BIN_FILE}
+        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/${TEST_UNIT_BIN_FILE} ${PROJECT_SOURCE_DIR}/run/
         DEPENDS robot2015_test_ELF
         COMMENT "Objcopying for making the ${HW_TEST_UNIT} hardware test"
     )
     set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${PROJECT_SOURCE_DIR}/run/${TEST_UNIT_BIN_FILE})
 
     add_custom_target(robot2015-test-prog
-        arm-none-eabi-objcopy -O binary robot2015_test_ELF ${TEST_UNIT_BIN_FILE}
+        ${ARM_OBJCOPY} -O binary robot2015_test_ELF ${TEST_UNIT_BIN_FILE}
```

Comment:
This objcopy should be removed since it's already handled in the `robot2015-test` target above.